### PR TITLE
Missing translation keys in enterprise fee menu 

### LINKED ISSUE #3722: Missing translation keys in enterprise fee menu
## Description
When using the enterprise fee menu the tax dropdown "Inherit from product" is not translated.

The whole dropdown for the column showing the type of fee isn't as well.

Same for the "search" placeholder.

## Steps to Reproduce
1. As an enterprise admin got to admin/enterprise_fees
2. Clic on "manage fees"
3. See the screen captured below

## Severity
bug-s3: a feature is broken but there is a workaround

## Your Environment
* Version used: v1.29
* Browser name and version: Firefox 66
* Operating System and version (desktop or mobile): Ubuntu 18.10
* OFN Platform instance where you discovered the bug: French production

### PR DESCRIPTION
**Pull request by amers185**
#### What? Why?
The enterprise fee management menu has two missing keys. Namely, the search text in search bar and the inherit from product option in tax category. 

Closes #3722
I have generated keys for both and put them in the en.yml file. After reviewing, both cases work. 

#### What should we test?

Check if the correct keys are accessed. 

#### Release notes

 Transifex will do its magic once everything is merged

FIxed

### CODE REVIEW

File: app/assets/javascripts/admin/enterprise_fees/controllers/enterprise_fees_controller.js.coffee [hunk 1]
```diff
@@ -1,6 +1,6 @@
 angular.module('admin.enterpriseFees').controller 'enterpriseFeesCtrl', ($scope, $http, $window, enterprises, tax_categories, calculators) ->
   $scope.enterprises = enterprises
-  $scope.tax_categories =  [{id: -1, name: "Inherit From Product"}].concat tax_categories
+  $scope.tax_categories =  [{id: -1, name: ('js.admin.enterprise_fees.inherit_from_product' | t) }].concat tax_categories
```
Comment (luisramos0) [inline app/assets/javascripts/admin/enterprise_fees/controllers/enterprise_fees_controller.js.coffee]: ah much better, now we can talk here :-)
the technique I use to fix these things is to look around in similar files, there are many points in the js code where things like this happen.
I would try this:
```suggestion
  $scope.tax_categories =  [{id: -1, name: "{{ 'js.admin.enterprise_fees.inherit_from_product' | t }}" }].concat tax_categories
```
and if it doesnt work, I'd use the t function:
```suggestion
  $scope.tax_categories =  [{id: -1, name: t('js.admin.enterprise_fees.inherit_from_product') }].concat tax_categories
```

**Comment by amers185** [comment]
Is there a way to do it without rebasing? Do I need to open a new PR to do that?

**Comment by Matt-Yorkley** [comment]
To squash you'll need to do an interactive rebase. You don't need a new PR though. After it's squashed you can do another `push` with the same branch using the `-f` flag to force the update.

**Comment by amers185** [comment]
@sauloperez Thank you! I have looked into git submodules and I am unsure about what commands to locate and  remove the .rbenv file. What you have any insight for me?

**Comment by amers185** [comment]
@sauloperez Thank you. I deleted the file you asked me to delete. Is there anything else you would like me to do?

**Comment by sauloperez** [comment]
Better now :clap: . You should squash the commits as suggested by @luisramos0 . As @Matt-Yorkley said you can do that with an interactive rebase and then forcing the push. You can read about it in https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history#squash-commits-together.

Don't hesitate to ask in Slack's [dev channel](dev channel of [our Slack team](https://openfoodnetwork.org/slack-invite).) if you need help.

**Comment by amers185** [comment]
Thank you. I squashed them all. Please let me know if need to do more. :)

**Comment by sauloperez** [comment]
Looks like the ineractive merge didn't work out quite well. There are a bunch of changes reverting things present in master and are unrelated to your PR.

I recommend you undo all changes with a `git reset --hard upstream/master` (you'll need to have upstream/master up-to-date) and then apply them again.

**Comment by luisramos0** [comment]
you probably need:
git checkout master
git pull upstream master
git checkout my_work
git rebase master

make sure you give your commit message a good description about the changes you made

**Comment by amers185** [comment]
After taking a deep-dive into my git history log, I realized I made some errors while rebasing. I have somehow ended up with two different versions of my_work, one of which sits in on github and the other one sits on local machine. 
image

image

They are at different points of the project, as such, that's why the changes made keep reverting. I am wondering about possible solutions for this but I just wanted to update you all. On another note, it is wonderful to have you back @luisramos0

**Comment by luisramos0** [comment]
:+1:

to make local like remote you do:
git checkout my_work
git reset --hard origin/my_work

to make remote like local you do:
git checkout my_work
git push -f origin my_work

**Comment by luisramos0** [comment]
this will work:

git checkout master
git reset --hard upstream/master
git checkout my_work
git reset --hard origin/my_work
git rebase master

and after resolving conflicts if any:
git push -f origin my_work

**Pull request closed without merging**